src/protobot_description/urdf/proto_protobot.urdf.xacro

```diff
@@ -73,4 +73,56 @@
         <material>Gazebo/Orange</material>
     </gazebo>
 
+    <gazebo>
+        <plugin name="imu_plugin" filename="libgazebo_ros_imu.so">
+            <alwaysOn>true</alwaysOn>
+            <bodyName>gps_imu_link</bodyName>
+            <topicName>imu/data</topicName>
+            <serviceName>imu_service</serviceName>
+            <gaussianNoise>0</gaussianNoise>
+            <updateRate>20.0</updateRate>
+        </plugin>
+    </gazebo>
+
+    <gazebo>
+        <plugin name="gps_controller" filename="libhector_gazebo_ros_gps.so">
+            <updateRate>30.0</updateRate>
+            <bodyName>gps_antenna_link</bodyName>
+            <topicName>gps/fix</topicName>
+            <frameId>gps_antenna_link</frameId>
+            <gaussianNoise>0</gaussianNoise>
+        </plugin>
+    </gazebo>
+
+    <gazebo reference="d435i_link">
+        <sensor type="depth" name="depth_camera">
+            <plugin name="camera_link_controller" filename="libgazebo_ros_openni_kinect.so">
+                <baseline>0.2</baseline>
+                <alwaysOn>true</alwaysOn>
+                <updateRate>1.0</updateRate>
+                <cameraName>camera</cameraName>
+                <imageTopicName>color/image_raw</imageTopicName>
+                <cameraInfoTopicName>color/camera_info</cameraInfoTopicName>
+                <depthImageTopicName>depth/image_raw</depthImageTopicName>
+                <depthImageInfoTopicName>depth/camera_info</depthImageInfoTopicName>
+                <pointCloudTopicName>depth/pointcloud</pointCloudTopicName>
+                <frameName>d435i_link</frameName>
+                <pointCloudCutoff>0.1</pointCloudCutoff>
+                <pointCloudCutoffMax>5</pointCloudCutoffMax>
+                <distortionK1>0.00000001</distortionK1>
+                <distortionK2>0.00000001</distortionK2>
+                <distortionK3>0.00000001</distortionK3>
+                <distortionT1>0.00000001</distortionT1>
+                <distortionT2>0.00000001</distortionT2>
+                <CxPrime>0</CxPrime>
+                <Cx>0</Cx>
+                <Cy>0</Cy>
+                <focalLength>0</focalLength>
+                <hackBaseline>0</hackBaseline>
+            </plugin>
+        </sensor>
+    </gazebo>
+
+    <!-- Implement t265_link -->
+
```

Comment:
Good opportunity to integrate this plugin https://github.com/pal-robotics/realsense_gazebo_plugin